src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
```

Comment:
You can format as here:
const { 
  name,
  age, 
  sex, 
  isMarried,
  partnerName } = person;